properties.schema

```diff
@@ -198,17 +197,17 @@
                 "default": "",
                 "inputType": "Text",
                 "validators": [],
-                "help": "Title text for this image",
+                "help": "Title text for the popup image",
                 "translatable": true
               },
               "_classes": {
                 "type": "string",
                 "required": false,
                 "default": "",
-                "title": "Classes",
+                "title": "CSS Classes",
```

Comment:
I'm not sure about this.  None of the other `_classes` properties are actually labelled as "CSS Classes", including the component level `_classes` in this same file.

Reply:
OK fair enough